bin/gftools-fix-isfixedpitch.py

```diff
@@ -0,0 +1,38 @@
+"""Fix a collection of fonts isfixedpitch flag"""
+from fontTools.ttLib import TTFont
+import argparse
+
+
+def fix_isFixedPitch(ttfont):
+
+    same_width = set([])
```

Comment:
Why not just `same_width = set()`?

Reply:
Sorry this was a demo script

Reply:
I guess we'll improve it in time. Was done over a google hangout.